Multithreading/src/Multithreading.java

```diff
@@ -0,0 +1,40 @@
+public class Multithreading {
+
+    public static void main(String[] args) throws InterruptedException {
+
+        //--------------------Homework---------------------------
+
+        Counter counter = new Counter();
+
+        Thread threadIncrement = new Thread(counter::increment);
+
+        Thread threadDecrement = new Thread(counter::decrement);
+
+        threadIncrement.start();
+        threadIncrement.join();
+        threadDecrement.start();
+        threadDecrement.join();
+
```

Comment:
This is actually not quite right: threads are running in sequence, not in parallel.